api/envoy/extensions/access_loggers/open_telemetry/v3alpha/logs_service.proto

```diff
@@ -18,9 +18,9 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: OpenTelemetry (gRPC) Access Log]
 
 // Configuration for the built-in *envoy.access_loggers.open_telemetry*
-// :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`. This configuration will
-// populate [opentelemetry.proto.collector.v1.logs.ExportLogsServiceRequest.resource_logs](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/collector/logs/v1/logs_service.proto).
-// OpenTelemetry [Resource](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/resource/v1/resource.proto)
+// :ref:`AccessLog <envoy_api_msg_config.accesslog.v4alpha.AccessLog>`. This configuration will
```

Comment:
i think this should be

```
ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
```
its causing the docs build to fail so its not clear why you set it to v4alpha from the diff

either way there are no v4alpha docs so i dont think there should be any links anywhere with that

Reply:
The auto-generation of protos keeps changing some unrelated protos and I got tired of reverting them so I just copied the documentation manually between v3/v4 and missed this change.
Fixed now.